lib/sacastats_web/templates/layout/app.html.heex

```diff
@@ -1,3 +1,5 @@
-<p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
-<p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
+<%= if not String.starts_with?(@conn.request_path, "/character/") do %>
```

Comment:
Why wrap the flash messages in this if? The outfit polls/pages definitely use them

Reply:
This is due to having the flash be below the sub-nav. I created a new one to be below it and that made for 2, so I removed the one that was above the sub-nav only when it exists (which rn is only for characters, this doesn't affect any outfit pages)

Reply:
Okay I see, I think in the future it would be better to have the flashes all in one place (preferably in the `app`/`live`/`root.html` templates). Maybe a solution to make sure they always show up below the subnav is have a blank subnav defined by default, and optionally populated by certain pages (like the character pages)?

Reply:
Ooo, I do like that, but my long-term solution was going to put them in a bootstrap toast https://getbootstrap.com/docs/4.3/components/toasts/

Not sure how they would exactly "trigger" yet, probably a listener on the text, but this way it will always be in the same spot and look a bit cleaner IMO.